setup.cfg

```diff
@@ -25,23 +25,23 @@ classifiers =
 [options]
 include_package_data = True
 install_requires =
```

Comment:
Can you add a comment here saying to complete the renovate `excludePackageNames` when someone add a new requirement ?